reference.md

```diff
@@ -763,7 +763,7 @@ Field Name | Presence | Type | Description
 
 ###### Query: 
 
-`https://www.example.com/gofs/1/en/wait_time?pickup_lat=45.60&pickup_lon=-73.30&brand_id=regular`
+`https://www.example.com/gofs/1/en/wait_time?pickup_lat=45.60&pickup_lon=-73.30&brand_id=regular_ride&brand_id=large_ride`
```

Comment:
@E-Baird 
Are you ok with the encoding of array in query string?

Reply:
I would prefer comma-separated values:

eg: `brand_id=regular_ride,large_ride`

Reply:
Ok, see my last commit.